lib/cog_api/fake/client.ex

```diff
@@ -84,6 +86,18 @@ defmodule CogApi.Fake.Client do
     Roles.revoke(endpoint, role, group)
   end
 
+  def rule_index(command, %Endpoint{}=endpoint) do
```

Comment:
Also, this will most certainly break the factories in Flywheel (once we use one for rules) unless we write some logic around it or make them all consistent